graphql/src/execution/execution.rs

```diff
@@ -410,6 +410,7 @@ pub async fn execute_root_selection_set<R: Resolver>(
             Arc::new(tokio::sync::Semaphore::new(1))
                 .acquire_owned()
                 .await
+                .unwrap()
```

Comment:
I don't know if it is a project standard to document why the `unwrap` can be used, should here have a comment as well?

Reply:
It is good practice, I've added a comment.